powerbi_setup/DOC Spyfish report.pbip

```diff
@@ -0,0 +1,14 @@
+{
+  "$schema": "https://developer.microsoft.com/json-schemas/fabric/pbip/pbipProperties/1.0.0/schema.json",
+  "version": "1.0",
+  "artifacts": [
+    {
+      "report": {
+        "path": "DOC Spyfish report.Report"

```

Comment:
I still think the report file itself is missing. Is there a different file that gets exported at the same time?